actioncable/lib/action_cable/server/broadcasting.rb

```diff
@@ -46,7 +39,7 @@ def initialize(server, broadcasting)
 
           def broadcast(message)
             server.logger.info "[ActionCable] Broadcasting to #{broadcasting}: #{message}"
-            server.broadcasting_redis.publish broadcasting, ActiveSupport::JSON.encode(message)
+            server.pubsub.broadcast broadcasting, ActiveSupport::JSON.encode(message)
```

Comment:
`pubsub` is just the instance of `ActionCable::SubscriptionAdapter::**`. https://github.com/rails/rails/pull/22950/files#diff-32900b7f3328eeea82a1443ed88197b0R50